Source:
Internal

Info:
https://www.gnu.org/software/gettext/manual/html_node/AM_005fGNU_005fGETTEXT_005fVERSION.html
The AM_GNU_GETTEXT_VERSION macro declares the version number of the GNU 
gettext infrastructure that is used by the package.
The use of this macro is optional; only the autopoint program makes use of 
it.

Status:
Need to determine if this patch has been sent upstream.

--- libgpg-error-1.12/configure.ac.orig	2010-10-26 17:13:48.000000000 +0800
+++ libgpg-error-1.12/configure.ac	2010-11-11 17:15:10.972945315 +0800
@@ -110,11 +110,13 @@
 #define GPG_ERR_ENABLE_ERRNO_MACROS 1
 ])
 
-
-# Note, that autogen.sh greps for the next line.
-AM_GNU_GETTEXT_VERSION([0.17])
-AM_GNU_GETTEXT([external])
-
+LTLIBINTL=
+AC_SUBST(LTLIBINTL)
+GETTEXT_PACKAGE=libgpg-error
+AC_SUBST(GETTEXT_PACKAGE)
+ALL_LINGUAS="cs de es fr hu it ja ko pl pt_BR ro ru sv vi zh_CN zh_HK zh_TW    "
+AM_GLIB_GNU_GETTEXT
+ 
 # Checks for header files.
 AC_HEADER_STDC
 AC_CHECK_HEADERS([stdlib.h locale.h])
